python/doc/theory/reliability_sensitivity/taylor_moments.rst

```diff
@@ -1,144 +1,106 @@
-Taylor variance decomposition
------------------------------
-
-| The Taylor variance decomposition (also referred as quadratic cumul method
-  or perturbation method) is a probabilistic approach designed to propagate
-  the uncertainties of the input variables :math:`\uX` through the model
-  :math:`h` towards the output variables :math:`\uY`. It enables to
-  access the central dispersion (expectation and variance) of the output
-  variables.
-| This method is based on a Taylor decomposition of the output variable
-  :math:`\uY` towards the :math:`\uX` random vectors around the mean
-  point :math:`\muX`. Depending on the order of the Taylor decomposition
-  (classically first order or second order), one can obtain different
-  formulas. For easiness of the reading, we first present the formulas
-  with :math:`n_Y=1` before the ones obtained for :math:`n_Y > 1`.
-| As :math:`Y=h(\uX)`, the Taylor decomposition around
-  :math:`\ux = \muX` at the second order yields to:
-
-| **Case** :math:`n_Y=1`
+.. _taylor_expansion_moments:
 
-.. math::
+Taylor expansion moments
+------------------------
 
-    Y = h(\muX) + < \underline{\nabla} h(\muX) , \: \uX - \muX> + \frac{1}{2}<<\underline{\underline{\nabla }}^2 h(\muX,\: \underline{\mu}_{\:X}),\: \uX - \muX>,\: \uX - \muX> + o(\Cov \uX)
+One way to evaluate the central dispersion (expectation and variance) of the variable :math:`\uY=h(\uX)`
+is to use the Taylor expansion of the function :math:`h: \Rset^d \rightarrow \Rset^q` at
+the mean point :math:`\vect{\mu} = \Expect{\uX}`.
 
-where:
+Depending on the order of the Taylor expansion (classically first or second order), we get different approximations
+of the mean and variance of :math:`Y`.
 
--  :math:`\muX = \Expect{\uX}` is the vector of the input variables at
-   the mean values of each component.
 
--  :math:`\Cov \uX` is the covariance matrix of the random vector
-   :math:`\uX`. The elements are the followings :
-   :math:`(\Cov \uX)_{ij} = \Expect{\left(X^i - \Expect{X^i} \right)\times\left(X^j - \Expect{X^j} \right)}`
+In the remainder, we denote by :math:`\Cov \uX`  the covariance matrix of :math:`\uX`, defined by:
 
--  :math:`\underline{\nabla} h(\muX) = \: ^t \left( \frac{\partial y}{\partial x^j}\right)_{\ux\: =\: \muX} = \: ^t \left( \frac{\partial h(\ux)}{\partial x^j}\right)_{\ux\: =\: \muX}`
-   is the gradient vector taken at the value :math:`\ux = \muX` and
-   :math:`j=1,\ldots,n_X`.
+.. math::
 
--  :math:`\underline{\underline{\nabla}}^2 h(\ux,\ux)` is a matrix. It
-   is composed by the second order derivative of the output variable
-   towards the :math:`i^\textrm{th}` and :math:`j^\textrm{th}`
-   components of :math:`\ux` taken around :math:`\ux = \muX`. It yields
-   to:
-   :math:`\left( \nabla^2 h(\muX,\muX) \right)_{ij} =\left( \frac{\partial^2 h(\ux,\ux)}{\partial x^i \partial x^j}\right)_{\ux\: =\: \muX}`
+    \Cov \uX = (c_{ij})_{ij}
```

Comment:
This equation has no meaning from my point of view.
```suggestion
    \Cov \uX = \mat{C}
```

Reply:
ok